docs/src/components/ComponentDoc/ComponentExample/ComponentExample.tsx

```diff
@@ -549,7 +552,7 @@ class ComponentExample extends React.Component<ComponentExampleProps, ComponentE
   }
 
   render() {
-    const { children, description, title } = this.props
+    const { children, description, title, excludedThemes } = this.props
```

Comment:
Docsite should document API - for that reason I would not hide examples based on theme capabilities.

In color palette PR, we had a long discussion about themes opting out/in for some library features and decided not to do that now as it can be tricky.

Reply:
agree to omit this `exclusion` functionality - in contrast, we should rather think about 'extending' approach, where there will be some base theme, and each new one will be able to include its own examples (like `inherit color` which we have for Teams theme). But, once again, it should be an approach that is driven by `extensibility`, not `exclusion` - thus I agree with suggestion to remove it